packages/avatar/src/index.js

```diff
@@ -1,7 +1,4 @@
-import Avatar from './react/index.js'
-import * as vars from './vars/index.js'
-import * as js from './js/index.js'
-import css from './css/index.js'
-
-export default Avatar
-export { vars, css, js }
+export { default } from './react/index.js'
+export * from './vars/index.js'
+export * from './js/index.js'
```

Comment:
This approach to export everything in these files to the same namespace feels a little unexpected.  Will there be naming overrides that happen unintentionally here?  

At the same time, I think it paves the way to a refactor I've been wanting to get to in the future:  `vars` feels like a pretty low-value grouping.  I think I'd rather just move vars into js.  And for packages with no vars and just js today, rename vars to js (more general).

Reply:
A shared namespace would concern me if these were larger packages honestly. Even util is tiny. This is one of the benefits of taking this monorepo approach. We're already renaming the groups when we import these in to other files for example in the `css/index.js` files I came across the following often `import { vars as iconVars } from '@pluralsight/ps-design-system-icon'` and most often the we were using `iconVars.widths`. I figured we could save ourselves some keystrokes and go with `import { widths as iconWidths } from '@pluralsight/ps-design-system-icon'`. 

As for the dropping vars for js. I can do that let me look over current usage and see if I see any potential gotchas.

Reply:
The vars -> js refactor definitely doesn't happen (maybe shouldn't, to keep the diffs smaller and our effort more focused) in this pr.